client/src/components/Attachment/AttachmentRelatedObjectsTable.js

```diff
@@ -0,0 +1,38 @@
+import LinkTo from "components/LinkTo"
+import Model from "components/Model"
+import React from "react"
+import { Table } from "react-bootstrap"
+
+const AttachmentRelatedObjectsTable = ({ relatedObjects }) => {
+  return (
+    <div id="related_objects">
+      {relatedObjects?.length > 0 ? (
```

Comment:
To prevent the "dangling else problem", reverse this condition, which makes the code more readable IMHO:
```javascript
diff --git a/client/src/components/Attachment/AttachmentRelatedObjectsTable.js b/client/src/components/Attachment/AttachmentRelatedObjectsTable.js
index aae4476fa..dff31a459 100644
--- a/client/src/components/Attachment/AttachmentRelatedObjectsTable.js
+++ b/client/src/components/Attachment/AttachmentRelatedObjectsTable.js
@@ -6,7 +6,9 @@ import { Table } from "react-bootstrap"
 const AttachmentRelatedObjectsTable = ({ relatedObjects }) => {
   return (
     <div id="related_objects">
-      {relatedObjects?.length > 0 ? (
+      {!relatedObjects?.length ? (
+        <em>No linked objects</em>
+      ) : (
         <Table striped hover responsive className="related_objects_table">
           <tbody>
             {relatedObjects.map(attachRelObj => (
@@ -24,8 +26,6 @@ const AttachmentRelatedObjectsTable = ({ relatedObjects }) => {
             ))}
           </tbody>
         </Table>
-      ) : (
-        <em>No linked objects</em>
       )}
     </div>
   )
```